qiskit/extensions/quantum_initialiser/__init__.py

```diff
@@ -0,0 +1 @@
+from .initialiser import Initialiser
```

Comment:
Can you revise this import so it mimics the mechanism used in `qiskit/extensions/standard/__init__.py` (which will also most likely include adding a `import qiskit.extensions.quantum_initialiser` to the main `qiskit/__init__.py`?

The idea would be to make `QuantumCircuit.initialise()` (and the rest of the functions) available when importing qiskit, ie making the `from qiskit.extensions.quantum_initialiser import Initialiser` lines on the examples not needed.